static/js/ducks/spectra.js

```diff
@@ -17,6 +17,8 @@ const PARSE_SOURCE_SPECTRUM_ASCII_OK =
 
 export const RESET_PARSED_SPECTRUM = "skyportal/RESET_PARSED_SPECTRUM";
 
+const ADD_TNS = "skyportal/ADD_TNS";
```

Comment:
Same here with spectrum. In fact, I'm not sure if this is a bug where multiple API calls have the same tag. In any case, better make them specific.

Reply:
yes good.